README.md

```diff
@@ -39,12 +39,16 @@ Commit your code regularly and use descriptive messages. This helps both you (in
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] What is the purpose of using _sessions_?
+      \*\*\* simple way to store data for individual users against a unique session ID.
 
-- [ ] What does bcrypt do to help us store passwords in a secure manner.
+* [ ] What does bcrypt do to help us store passwords in a secure manner.
```

Comment:
bcrypt is a password hashing function that changes the value of the password, encrypting each character.